code/modules/mob/living/carbon/xenomorph/mutators/strains/boiler/trapper.dm

```diff
@@ -64,30 +64,30 @@
 		temp_movespeed_usable = FALSE
 
 		if (isXeno(bound_xeno))
-			var/mob/living/carbon/Xenomorph/X = bound_xeno
-			X.speed_modifier -= temp_movespeed_amount
-			X.recalculate_speed()
+			var/mob/living/carbon/Xenomorph/xeno = bound_xeno
+			xeno.speed_modifier -= temp_movespeed_amount
+			xeno.recalculate_speed()
 			addtimer(CALLBACK(src, .proc/remove_speed_buff), temp_movespeed_duration)
 
-/datum/behavior_delegate/boiler_trapper/ranged_attack_additional_effects_target(atom/A)
-	if (!ishuman(A))
+/datum/behavior_delegate/boiler_trapper/ranged_attack_additional_effects_target(atom/target_atom)
+	if (!ishuman(target_atom))
 		return
 	if (!istype(bound_xeno))
 		return
 
-	var/mob/living/carbon/human/H = A
+	var/mob/living/carbon/human/target_human = target_atom
 	var/datum/effects/boiler_trap/found = null
-	for (var/datum/effects/boiler_trap/F in H.effects_list)
+	for (var/datum/effects/boiler_trap/F in target_human.effects_list)
 		if (F.cause_data?.resolve_mob() == bound_xeno)
 			found = F
 			break
```

Comment:
👀 what's all this then

Reply:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA